moveit_experimental/moveit_jog_arm/src/jog_calcs.cpp

```diff
@@ -413,8 +421,16 @@ void JogCalcs::insertRedundantPointsIntoTrajectory(trajectory_msgs::JointTraject
   }
 }
 
-void JogCalcs::lowPassFilterPositions(sensor_msgs::JointState& joint_state)
+void JogCalcs::lowPassFilterPositions(sensor_msgs::JointState& joint_state, JogArmShared& shared_variables)
 {
+  // Check if a new filter coefficient was supplied
+  if (!shared_variables.filter_coefficient_queue.empty())
+  {
+    double new_coefficient;
+    shared_variables.filter_coefficient_queue.pop(new_coefficient);
+    updateFilterCoeff(new_coefficient);
+  }
```

Comment:
[nit] this should probably pop all new values to get the last one sent.  To do that check the size of the queue and pop that number of times.  Even though this is unlikely, it is probably the behavior you want?